### ISSUE TITLE
Segmantation Fault (core dumped)

### ISSUE THREAD

**Issue by 1215481871** [zh217/torch-dct#19]
When I import and use this package, it sometimes appears the segmentation fault (core dumped）problem.  Has any one meet this kind of problem before?

**Comment by TianhaoFu** [comment]
Hi, I came acorss same problem. It seems because of torch.rfft().

Did you fixed it ?

**Comment by 1215481871** [comment]
Sorry, I haven't. It still happens sometimes. What's wrong with torch.rfft()? Can you explained it or how I can fix this problem?

**Comment by TianhaoFu** [comment]
maybe you can see this：
[(https://github.com/pytorch/pytorch/issues/25640.)]

BTW, so now you run your code many times in order to makes it run successfully?

**Comment by 1215481871** [comment]
Yes, I'm still struggling on this problem. Have you fixed it yet?
